src/legacy/core_plugins/data/public/actions/filters/brush_event.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { get, last } from 'lodash';
+import moment from 'moment';
+import {
+  esFilters,
+  IFieldType,
+  RangeFilterParams,
+  IIndexPattern,
+} from '../../../../../../plugins/data/public';
+import { deserializeAggConfig } from '../../search';
+// should be removed after moving into new platform plugins data folder
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { getIndexPatterns } from '../../../../../../plugins/data/public/services';
+
+export interface BrushEvent {
+  data: {
+    ordered: {
+      date: boolean;
+    };
+    series: Array<Record<string, any>>;
+    indexPattern: IIndexPattern;
```

Comment:
this is not required, index pattern is retrieved from column meta info

Reply:
removed